components/notifications-client/builder/compliance.go

```diff
@@ -119,6 +119,13 @@ func collectNotificationsControl(control *inspec.Control, agg *complianceReportS
 	failed := 0
 	total := 0
 
+	if control.RemovedResultsCounts != nil {
+		failed = int(control.RemovedResultsCounts.Failed)
+		passed = int(control.RemovedResultsCounts.Passed)
+		skipped = int(control.RemovedResultsCounts.Skipped)
+		total = failed + passed + skipped
+	}
+
```

Comment:
Adding the counts from the truncated tests. This was the original fix for this PR.